tests/run/value-class-extractor.scala

```diff
@@ -6,12 +6,12 @@ object NonNullChar {
   @inline final val None = new NonNullChar(0.toChar)
 }
 
-final class SomeProduct extends Product3[String, Int, List[String]] {
+final class SomeProduct /*extends Product3[String, Int, List[String]]*/ {
   def canEqual(x: Any) = x.isInstanceOf[SomeProduct]
   def _1 = "abc"
   def _2 = 5
   def _3 = List("bippy")
-  def isEmpty = false
+  def isDefined = true
```

Comment:
We want to try maintaining tests compatible with `scalac` as much as possible. It's better to maintain `isEmpty` method.